shared/domains/chelonia/db.js

```diff
@@ -15,21 +15,21 @@ sbp('sbp/selectors/unsafe', ['chelonia/db/get', 'chelonia/db/set', 'chelonia/db/
 
 const dbPrimitiveSelectors = process.env.LIGHTWEIGHT_CLIENT === 'true'
   ? {
-      'chelonia/db/get': function (key): Promise<*> {
+      'chelonia/db/get': function (key: string): Promise<string | void> {
         const id = sbp('chelonia/db/contractIdFromLogHEAD', key)
-        return Promise.resolve(id ? sbp(this.config.stateSelector).contracts[id]?.HEAD : null)
+        return Promise.resolve(id ? sbp(this.config.stateSelector).contracts[id]?.HEAD : undefined)
       },
-      'chelonia/db/set': function (key, value): Promise<void> { return Promise.resolve(value) },
-      'chelonia/db/delete': function (): Promise<void> { return Promise.resolve() }
+      'chelonia/db/set': function (key: string, value: string): Promise<string> { return Promise.resolve(value) },
```

Comment:
Why does this return signature return a `Promise<string>`, while the overwritten `'chelonia/db/set'` return `Promise<void>`? They should be consistent...